docs/lib/restapi/fragments/flutter/getting-started/10_preReq.md

```diff
@@ -0,0 +1,5 @@
+* [Install and configure Amplify CLI](https://docs.amplify.aws/cli/start/install)
+* A Flutter application targeting Flutter SDK >= 1.20 with Amplify libraries integrated
+    * An iOS configuration targeting at least iOS 13.0
```

Comment:
Is this accurate? iOS 13.0 for Rest API.

Reply:
Thanks it just needs to be 11.0 , changed it to that.